Code/MinimalLib/jswrapper.cpp

```diff
@@ -218,19 +232,6 @@ std::string generate_aligned_coords_helper(JSMol &self,
   return self.generate_aligned_coords(templateMol, param.as<std::string>());
 }
 
-std::string parse_morgan_fp_param(unsigned int radius, unsigned int fplen,
```

Comment:
Why did this get pulled out? 
Doesn't seem connected to the PR, but if this is redundant code that's being removed you probably should mention it in the PR description

Reply:
That will go in its own PR; sorry for the confusion.